src/inspector/runtime_agent.cc

```diff
@@ -0,0 +1,43 @@
+#include "runtime_agent.h"
+
+#include "env-inl.h"
+#include "inspector_agent.h"
+
+namespace node {
+namespace inspector {
+namespace protocol {
+
+RuntimeAgent::RuntimeAgent(Environment* env)
+  : enabled_(false)
+  , env_(env) {}
```

Comment:
I believe the convention was to have the comma on the previous line (I apologize if the convention had changed)

Reply:
I put them to one line - the same way as in tracing_agent.cc